.github/workflows/go-test.yaml

```diff
@@ -43,5 +43,6 @@ jobs:
           # let's wait for operator pod to be restart
           kubectl -n rook-ceph wait --for=delete pod/$POD --timeout=100s
           tests/github-action-helper.sh wait_for_operator_pod_to_be_ready_state_default
+          kubectl rook_ceph debug start rook-ceph-osd-0
```

Comment:
What tests can be added? Probably to wait for the debug pod to start, then also stop the debug pod and wait for it to stop?

Reply:
added